rust/operator-binary/src/airflow_controller.rs

```diff
@@ -1333,18 +1386,20 @@ fn add_authentication_volumes_and_volume_mounts(
     authentication_config: &Vec<AirflowAuthenticationConfigResolved>,
     cb: &mut ContainerBuilder,
     pb: &mut PodBuilder,
-) {
+) -> Result<()> {
     // TODO: Currently there can be only one AuthenticationClass due to FlaskAppBuilder restrictions.
     //    Needs adaptation once FAB and airflow support multiple auth methods.
     // The checks for max one AuthenticationClass and the provider are done in crd/src/authentication.rs
     for config in authentication_config {
         if let Some(auth_class) = &config.authentication_class {
             match &auth_class.spec.provider {
                 AuthenticationClassProvider::Ldap(ldap) => {
-                    ldap.add_volumes_and_mounts(pb, vec![cb]);
+                    ldap.add_volumes_and_mounts(pb, vec![cb])
+                        .context(VolumeAndMountsSnafu)?;
                 }
-                AuthenticationClassProvider::Tls(_) | AuthenticationClassProvider::Static(_) => {}
```

Comment:
Nit: It would be great if we could have the variants explicit. But that's arguable so optional ;)

Reply:
Fixed in eda48f4763a061eaf2679fbb6c6482afd363b618